store/postgres/src/entities.rs

```diff
@@ -649,9 +676,13 @@ impl<'a> Connection<'a> {
                 "state" => format!("{:?}", schema.state)
             );
             let start = Instant::now();
-            let storage = storage.migrate(&self.conn, logger, &schema)?;
-            let needs_migrating = storage.needs_migrating();
-            self.cache.borrow_mut().insert(subgraph.clone(), storage);
+            // Do the actual migration, and return an updated storage
+            // object, something like
+            //
+            // let storage = storage.migrate(&self.conn, logger, &schema)?;
+            // let needs_migrating = storage.needs_migrating();
+            // self.cache.borrow_mut().insert(subgraph.clone(), Arc::new(storage));
+            //
```

Comment:
What's up with the commented out code?

Reply:
The migration machinery was quite painful to get right, so I want to keep it around, even if it's not being used at the moment. This code would need more fixing up than I like to keep compiling as `storage.migrate` took ownership of `storage` and possibly returned a new one. That doesn't work with how we cache anymore and needs to be rethought.

Reply:
You could put `unreachable` somewhere to indicate that it's ok for this code to not be working.

Reply:
Did that and a bunch of `allow` to make rustc not complain